subxt/Cargo.toml

```diff
@@ -32,23 +39,32 @@ scale-info = "2.0.0"
 scale-value = "0.6.0"
 scale-bits = "0.3"
 scale-decode = "0.4.0"
-futures = { version = "0.3.13", default-features = false }
+futures = { version = "0.3.13", default-features = false, features = ["std"] }
 hex = "0.4.3"
 jsonrpsee = { version = "0.16", optional = true, features = ["jsonrpsee-types"] }
 serde = { version = "1.0.124", features = ["derive"] }
 serde_json = { version = "1.0.64", features = ["raw_value"] }
 thiserror = "1.0.24"
 tracing = "0.1.34"
 parking_lot = "0.12.0"
+frame-metadata = "15.0.0"
+derivative = "2.2.0"
 
 subxt-macro = { version = "0.25.0", path = "../macro" }
 subxt-metadata = { version = "0.25.0", path = "../metadata" }
 
-sp-core = { version = "7.0.0", default-features = false  }
-sp-runtime = "7.0.0"
+# Provides some deserialization, types like U256/H256 and hashing impls like twox/blake256:
+impl-serde = { version = "0.4.0" }
+primitive-types = { version = "0.12.0", default-features = false, features = ["codec", "scale-info", "serde"] }
+sp-core-hashing = "6.0.0"
```

Comment:
Also here if you import `twox-hash`, you already have `blake2` below and this can go. Would mean no more `sp-std`...

Reply:
Ooh I like it! Thanks for the idea!